composer.json

```diff
@@ -42,20 +42,22 @@
         }
     ],
     "require": {
-        "php": ">=7.0.0"
+        "php": ">=7.0.0",
+        "functional-php/pattern-matching": "~0"
     },
     "require-dev": {
         "phpunit/phpunit": "~6",
-        "squizlabs/php_codesniffer": "~3"
+        "friendsofphp/php-cs-fixer": "~2"
```

Comment:
It seems that we're going to replace `PHP_CodeSniffer` with `PHP-CS-Fixer`.

It there any reason to make this change?